WordImpress/Give-Addon-Boilerplate
# Add development tools

Resolves #16 #17

## Description


This PR adds development tools. I also formatted the existing code.

## Affects


I upgrade most of tools and removed which react related package which we can add in another issue after discussion.


## Pre-review Checklist

## Testing Instructions
- Run `npm i`
- Run `npm run dev`

For testing, you can add a testing commit to check if tools apply of your commit or not.
